### PR TITLE
changelog update Ruby SDK

### PR DESCRIPTION
**Pull request by LauraDuRy**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/changelog/(entries)/2024-01-19.markdoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: changelog
+title: New Ruby SDK release
+date: 2024-01-19
+cover:
+---
+
+We’ve just released a new Ruby SDK with fixes for the database service.
```
Comment (eldadfux) [inline src/routes/changelog/(entries)/2024-01-19.markdoc:8]:
```suggestion
We’ve just released a new Ruby SDK with fixes for the databases service.
```

File: src/routes/changelog/(entries)/2024-01-19.markdoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+What's changed? 
+
+- Fix null parameters being passed to API
```
Comment (eldadfux) [inline src/routes/changelog/(entries)/2024-01-19.markdoc:12]:
```suggestion
- Fix null parameters being passed to the API
```

File: src/routes/changelog/(entries)/2024-01-19.markdoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- Fix `require` in documentation
+
+View the [pull request](https://github.com/appwrite/sdk-for-ruby/compare/10.1.0...10.1.1) on GitHub for the source code.
```
Comment (eldadfux) [inline src/routes/changelog/(entries)/2024-01-19.markdoc:15]:
```suggestion
View the [changes](https://github.com/appwrite/sdk-for-ruby/compare/10.1.0...10.1.1) on GitHub.
```

File: src/routes/changelog/(entries)/2024-01-19.markdoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+date: 2024-01-19
+cover:
+---
+
+We’ve just released a new Ruby SDK with fixes for the database service.
+
+What's changed? 
+
+- Fix null parameters being passed to API
+- Fix `require` in documentation
```
Comment (eldadfux) [inline src/routes/changelog/(entries)/2024-01-19.markdoc:13]: Not sure what this means

**Review by eldadfux — CHANGES_REQUESTED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request closed without merging**